### ISSUE TITLE
[Request] Some interplay audio format from Stonekeep

### ISSUE THREAD

**Issue by Feathered-Serpent** [vgmstream/vgmstream#1402]
Within the CD of the Interplay game [Stonekeep ](https://en.wikipedia.org/wiki/Stonekeep)there is a file called VOICES without extension:
![grafik](https://github.com/vgmstream/vgmstream/assets/72162369/45920206-7da0-4dcc-9db4-c2410b6707e8)

I looped through all supported extensions and renamed the file and tried to show the metadata with -m switch. Only on those formats I got any results:
```
Input file: voices.int
metadata for voices.int
sample rate: 48000 Hz
channels: 2
stream total samples: 84004931 (29:10.103 seconds)
encoding: Little Endian 16-bit PCM
layout: interleave
interleave: 0x200 bytes
metadata from: PS2 .int raw header
bitrate: 1536 kbps
play duration: 84004931 samples (29:10.103 seconds)

Input file: voices.raw
metadata for voices.raw
sample rate: 44100 Hz
channels: 2
stream total samples: 84004931 (31:44.874 seconds)
encoding: Little Endian 16-bit PCM
layout: interleave
interleave: 0x2 bytes
metadata from: PC .raw raw header
bitrate: 1411 kbps
play duration: 84004931 samples (31:44.874 seconds)

Input file: voices.snds
metadata for voices.snds
sample rate: 48000 Hz
channels: 2
stream total samples: 336019726 (116:40.411 seconds)
encoding: Heavy Iron .snds 4-bit IMA ADPCM
layout: flat
metadata from: PC .snds raw header
bitrate: 384 kbps
play duration: 336019726 samples (116:40.411 seconds)

Input file: voices.wavm
metadata for voices.wavm
sample rate: 44100 Hz
channels: 2
stream total samples: 298684198 (112:52.884 seconds)
encoding: XBOX 4-bit IMA ADPCM
layout: flat
metadata from: Xbox .wavm raw header
bitrate: 396 kbps
play duration: 298684198 samples (112:52.884 seconds)

Input file: voices.wp2
metadata for voices.wp2
sample rate: 48000 Hz
channels: 4
stream total samples: 42002465 (14:35.051 seconds)
encoding: Little Endian 16-bit PCM
layout: interleave
interleave: 0x200 bytes
metadata from: PS2 .int raw header
bitrate: 3072 kbps
play duration: 42002465 samples (14:35.051 seconds)
```

When exporting those audio streams, I get several different sizes with the wave files:
![grafik](https://github.com/vgmstream/vgmstream/assets/72162369/5d41b407-062a-4c7a-bfd3-fed868445b6b)

Upon trying to play it, I can hear lots and lots of noise, but also voices within that noise, which are obviously speed up in some way. So the audio data is there. Any idea what I could do? I also assume that there are multiple audio streams inside, as I assume the file contains all spoken audio from within the game.

[Audio File to download](https://dragonswatch.silentstrider.net/sharing/O397HJeMz) (320MB, just tell me when I can delete the share destination)
[The master file, in case it's important](https://dragonswatch.silentstrider.net/sharing/LMWE1zIOX)

**Comment by bnnm** [comment]
Should work if you unzip this file near VOICES and open VOICES (use `-s` / `-S` to select subsongs). Sample rate is assumed so open and tweak the .txth as needed: [txth.zip](https://github.com/vgmstream/vgmstream/files/12445797/txth.zip)

txt for posterity:
```
subsong_count = @0x00

#04 name table start
base_offset = @0x08 #data table start
#0x0c: num strings

# read offset from data table
subsong_spacing = 0x04
base_offset = @0x00
subsong_spacing = 0x00

# entry
start_offset = @0x00 # - base_offset #absolute offset to data
data_size = @0x04
channels = @0x08
#0c: extra table entries (may be null)
#10: string size
#14: block size?
#18: voice line string
#xx: extra entries (0x00: ? + 0x02: N entries of size 0x02)
#xx: data

codec = PCM8_U
sample_rate = 36000
num_samples = data_size
```

**Comment by Feathered-Serpent** [comment]
Woah, that seems to work great! 36000 seems to be correct, because I tried some other sample rates, and they didn't sound right.

There is but one other thing:
![grafik](https://github.com/vgmstream/vgmstream/assets/72162369/894f887f-bdf3-4d23-98a6-ff9b072b532c)
There are 28 files in the end, which not only have more than one channel, but they're also need to seemingly played at 10% speed to have the correct audio. Or is this something sample rate related as well?
E.g. 1222 is a song sung from NPCs in the game I remembered upon hearing.
